crates/hash/src/hash.rs

```diff
@@ -74,7 +81,7 @@ impl AsMut<[u8]> for Inner {
 }
 
 impl std::fmt::Display for Inner {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
If we're bringing these into scope, shall we do the same for `Display`?

Reply:
Didn't mean to do that, I'll put it back.